gomplate.md

```diff
@@ -0,0 +1,5908 @@
+
+
+## `base64.Encode`
+
+Encode data as a Base64 string. Specifically, this uses the standard Base64 encoding as defined in [RFC4648 &sect;4](https://tools.ietf.org/html/rfc4648#section-4) (and _not_ the URL-safe encoding).
+
+Usage
+
+```go
+base64.Encode input
+```
+```go
+input | base64.Encode
+```
+
+Arguments
+
+| name | description |
+|------|-------------|
+| `input` | _(required)_ The data to encode. Can be a string, a byte array, or a buffer. Other types will be converted to strings first. |
+
+Examples
+
+```console
+$ gomplate -i '{{ base64.Encode "hello world" }}'
+aGVsbG8gd29ybGQ=
+```
+```console
+$ gomplate -i '{{ "hello world" | base64.Encode }}'
+aGVsbG8gd29ybGQ=
+```
```

Comment:
The examples should not be using the gomplate command line

```suggestion
```console
{{ base64.Encode "hello world" }} # GVsbG8gd29ybGQ=
```
```
```